plugins/wazuh_check_updates/server/services/userPreferences/updateUserPreferences.ts

```diff
@@ -0,0 +1,30 @@
+import { SAVED_OBJECT_USER_PREFERENCES } from '../../../common';
+import { UserPreferences, UsersPreferences } from '../../../common/types';
+import { getSavedObject, setSavedObject } from '../savedObject';
+
+export const updateUserPreferences = async (
+  username: string,
+  preferences: UserPreferences
+): Promise<UserPreferences> => {
+  try {
+    const usersPreferences =
+      ((await getSavedObject(SAVED_OBJECT_USER_PREFERENCES)) as UsersPreferences)?.users || [];
+
+    const userPreferences = usersPreferences?.find((up) => up.username === username) || {};
```

Comment:
**suggestion**: replace `up` variable name as possible.

Reply:
The name was changed